homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -89,12 +87,23 @@ def _get_2ch(self):
             in (IncomingAudioFormat.PCM, IncomingAudioFormat.ANALOGUE_DIRECT, None)
         )
 
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        """Return if the entity should be enabled when first added to the entity registry."""
+        return self._state.zn == 1
```

Comment:
Why not enable the other zones ?

Reply:
Other zones are very uncommonly used, and of quite limited features